src/chainstate/burn/operations/leader_block_commit.rs

```diff
@@ -394,6 +394,18 @@ impl LeaderBlockCommitOp {
                 previous_is_burn && output_addr.is_burn()
             })
     }
+
+    pub fn spent_txid(&self) -> Txid {
+        self.input.0.clone()
```

Comment:
Does this need to clone(), or could we make do with returning the `&Txid` here?

Reply:
Fixed in 9c6035c